components/usage/pkg/db/cost_center.go

```diff
@@ -263,3 +263,50 @@ func (c *CostCenterManager) ListLatestCostCentersWithBillingTimeBefore(ctx conte
 
 	return results, nil
 }
+
+func (c *CostCenterManager) ResetUsage(ctx context.Context, cc CostCenter) (CostCenter, *Usage, error) {
+	if cc.BillingStrategy != CostCenter_Other {
+		return CostCenter{}, nil, fmt.Errorf("cannot reset usage for Billing Strategy %s for Cost Center ID: %s", cc.BillingStrategy, cc.ID)
+	}
+
+	entity, _ := cc.ID.Values()
+
+	var spendingLimit int32
+	switch entity {
+	case AttributionEntity_Team:
+		spendingLimit = c.cfg.ForTeams
+	case AttributionEntity_User:
+		spendingLimit = c.cfg.ForUsers
+	default:
+		return CostCenter{}, nil, fmt.Errorf("cannot reset usage for unknown attribution entity ID: %s", cc.ID)
+	}
```

Comment:
no need to obtain the default spending limits as the cost center already has one

Reply:
See https://github.com/gitpod-io/gitpod/pull/14249#discussion_r1007691049 for reasoning.